tests/vcs/git/GitCommitBearTest.py

```diff
@@ -267,6 +269,136 @@ def test_body_checks(self):
                          [])
         self.assertTrue(self.msg_queue.empty())
 
+    def test_check_issue(self):
+        # Commit with no remotes configured
+        self.git_commit('Shortlog\n\n'
+                        'First line, blablablablablabla.\n'
+                        'Another line, blablablablablabla.\n'
+                        'Closes #01112')
+        self.assertEqual(self.run_uut(body_close_issue=True), [])
+
+        # Adding BitBucket remote for testing
+        self.run_git_command('remote', 'add', 'test',
+                             'https://bitbucket.com/user/repo.git')
+
+        # Unsupported Host
```

Comment:
you should maybe say in the comment before that BitBucket is the unsupported host :)